landing-page/content/common/release-notes.md

```diff
@@ -68,7 +68,7 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 ```
 ## 0.13.2 Release Notes
 
-Apache Iceberg 0.13.2 was released on June XXth, 2022.
+Apache Iceberg 0.13.2 was released on June 13th, 2022.
```

Comment:
The official release announcement was on `15 June 2022 20:41:59 CEST`

Reply:
Are you refering to the mail list announcement ? https://lists.apache.org/thread/rgx2q5kzzcgorhv494f37pstpbp5f6cy

updating it to 15th, thanks @pvary !